packages/types/src/index.ts

```diff
@@ -381,6 +381,8 @@ export type Fare = {
  * http://otp-docs.ibi-transit.com/api/json_Itinerary.html
  */
 export type Itinerary = {
+  co2?: number;
+  co2VsBaseline?: number;
```

Comment:
Is this used anywhere?

Reply:
The value is used in OTP-RR in an un-typescripted file, so at some point if we typescript that file we'll be grateful for this property being here.